aerogear/aerogear-unifiedpush-server
# Aerogear 8829 8831 apns token

## Motivation
Add support for APNS token signed push messages

## How
 * A second iOS variant type was created, iOSTokenVariant.
 * A second pushy sender was created
 * A common "IsAPNSVariant" interface was created for general APNS connectivity
 * JMS queues are unaffected
 * Tests were written as necessary 

## Verification Steps
 0.  Setup a APNS token project in Apple's dev console
 1. Create a variant using the RESTful API
 2. Verify that it appears in the admin-ui console; no work was done, but this shows it is ther
 3. Configure and register a mobile application using this variant
 4. Use the push sender API to send a message

### Verification notes : passos and I can provide the necessary help getting this set up; operator and MDC will handle the user facing bits.
